gcloud/datastore/client.py

```diff
@@ -441,8 +441,38 @@ def transaction(self):
 
     def query(self, **kwargs):
         """Proxy to :class:`gcloud.datastore.query.Query`.
-
```

Comment:
[PEP 257[(https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) requires the blank line.